tools/src/publish/bin/bump-package.ts

```diff
@@ -0,0 +1,47 @@
+import { readFileSync, writeFileSync } from 'node:fs';
+import { join } from 'node:path';
+import yargs from 'yargs';
+import { hideBin } from 'yargs/helpers';
+import { parseVersion } from '../../utils';
+import type { BumpOptions } from '../types';
+
+const argv = yargs(hideBin(process.argv))
+  .options({
+    nextVersion: {
+      type: 'string',
+    },
+    verbose: { type: 'boolean' },
+  })
+  .coerce('nextVersion', parseVersion).argv;
+
+const {
+  nextVersion: version,
+  verbose,
+  directory = process.cwd(),
+} = argv as BumpOptions;
+// Updating the version in "package.json"
+const packageJsonFile = join(directory, 'package.json');
+try {
+  const packageJson = JSON.parse(readFileSync(packageJsonFile).toString());
+  if (version != null) {
+    if (packageJson.version === version) {
+      console.info(`Package version is already set to ${version}.`);
+      process.exit(0);
+    }
+
+    console.info(
+      `Updating ${packageJsonFile} version from ${packageJson.version} to ${version}`,
+    );
+    writeFileSync(
+      packageJsonFile,
+      JSON.stringify({ ...packageJson, version }, null, 2),
+    );
+    process.exit(0);
+  }
+  // @TODO
+  console.info('Implement autodetect version bump');
+  process.exit(1);
```

Comment:
the log explains it already. 🫣

Reply:
Oh, that's you meant it :sweat_smile: An info log is different from a code comment, though. If I saw `Implement autodetect version bump` in the output logs, I wouldn't understand how it relates to the error code, it sounds like something that's being executed, not something that isn't implemented and therefore the process will fail.

This would be much clearer:

```suggestion
  // @TODO: Implement autodetect version bump
  console.info('Autodetecting version bump not yet implemented, exiting with error');
  process.exit(1);
```